💄 Component/header 1차 PR 요청

syon013 opened the pull request.

syon013 committed:
💄 Header PNG, SVG 파일 추가

- Header에 들어갈 아이콘 관련 PNG 파일과 SVG 파일 추가

syon013 committed:
📝 README 프로젝트 완료 날짜 수정

syon013 committed:
🚧 Header Component 진행 중

syon013 committed:
Merge branch 'master' of https://github.com/ryushin01/goobne into component/header

syon013 committed:
💄 Header Component 1차 PR 요청

- UI 완성 및 SubMenu on/off 기능 추가

syon013 committed:
🎨 Logo h1 태그 추가

ryushin01 commented on src/components/Header/Header.jsx:
```diff
@@ -0,0 +1,214 @@
+
+  // function
+  /**
+   * @param header '주문하기' 에 마우스 hover 시 SubMenu를 Opne 시키도록 하기위한 함수
```
@param은 파라미터가 어떤 데이터 타입으로 들어오는지 설명하기 위한 것이기에 함수를 설명하는 문구는  * 뒤에 쓰시면 됩니다.

syon013 replied:
@param 삭제 완료 했습니다.

ryushin01 commented on src/components/Header/Header.jsx:
```diff
@@ -0,0 +1,214 @@
+
+  // function
+  /**
+   * @param header '주문하기' 에 마우스 hover 시 SubMenu를 Opne 시키도록 하기위한 함수
```
오타가 있네요. Opne > Open

syon013 replied:
오타 수정 완료

ryushin01 commented on src/components/Header/Header.jsx:
```diff
@@ -0,0 +1,214 @@
+  return (
+    <HeaderContainer>
+      <HeaderInnerWrap>
+        <HeaderLogo
```
div태그인데 onClick을 주는건 적절하지 않습니다.

syon013 replied:
Link 태그로 수정 완료 했습니다.

ryushin01 commented on src/components/Header/Header.jsx:
```diff
@@ -0,0 +1,214 @@
+  height: 110px;
+  top: 0;
+  z-index: 99;
+  background-color: #fbf5f0;
```
이 컬러값은 한번만 사용하나요?

syon013 replied:
공통 theme로 수정 완료 했습니다.

syon013 committed:
🎨 리뷰 지적사항 수정

syon013 committed:
🎨 Header Component 수정

- 기존 함수 사용해서 Open -> Close 를 하는 방식으로 subMenu Open

- 조건부 랜더링에 의해서 화면 흔들림 현상 버그 발생

- display : none -> flex 로 하는 방식으로 css 수정 및 함수 삭제
